### ISSUE TITLE
"Requests to the server have been blocked by an extension" blocking download button on some deviantART pages

### ISSUE THREAD

**Issue by qwerty12** [uBlock-LLC/uBlock#70]
With Chromium 35.0.1916.153 (274914) and µBlock 0.2.1.2, going to http://satya164.deviantart.com/art/Conky-Google-Now-366545753, for example, will show the "Requests have been blocked" message as in the screenshot and cover the download box. This does not happen on all deviantART pages though (http://zagortenay333.deviantart.com/art/Conky-Harmattan-426662366 seems to be OK) and sometimes I won't even see the "Requests have been blocked" message - instead, I simply won't be able to click on the download button save for a tiny, tiny area around the top-left edge of the button. Using the Developer tools to delete `<div class="dp-ad-chrome atf-right-300x250 dp-ad-visible" style="top: 13px; left: 1035px;">` lets me workaround it.

Apologies if this is already covered by another bug. Thanks.

Hide placeholders of blocked elements: On
Enabled filter lists:

```
Your filters: 57 used out of 57
µBlock filters: 7 used out of 7
Long-lived malware domains (malwaredomains.com): 2,199 used out of 2,204
Malware domains (malwaredomains.com): 11,699 used out of 14,025
Malware Domain List (malwaredomainlist.com): 1,348 used out of 1,348
Spam404 (spam404bl.com): 4,143 used out of 4,143
EasyList (adblockplus.org): 18,852 used out of 18,852
EasyPrivacy (adblockplus.org): 9,072 used out of 9,075
Fanboy’s Annoyance List (adblockplus.org): 2,776 used out of 2,776
Fanboy’s Enhanced Tracking List (fanboy.co.nz): 703 used out of 703
Fanboy’s Anti-Facebook (see warning inside list) (fanboy.co.nz): 51 used out of 51
Anti-Adblock Killer | Filters for Adblockers (raw.githubusercontent.com): 247 used out of 247
```

"My" filters:

```
||platform.twitter.com/widgets.js$third-party
||gravatar.com^$third-party

# Blacklist maintained by HTTP Switchboard
# For those domain names which are not found in other blacklists

2mdn.net            # "2mdn.net is a domain used by Doubleclick which is an advertising company..."
aad73c550c.se       # Related to adrotator.se which is itself blacklisted
acxiom-online.com   # Wikipedia: "Acxiom Corporation is a marketing technology and services company".
adextent.com        # "We are an advertising technology company - we build technologies that improve ads performance"
adgear.com          # "AdGear is an online advertising technologies company based in Montreal, Canada"
adnxs.com           # "Adnxs.com is run by AppNexus, a company that provides technology, data and analytics to help companies buy and sell online display advertising" (Ref.: http://www.theguardian.com/technology/2012/apr/23/adnxs-tracking-trackers-cookies-web-monitoring)
adobetag.com        # "Adobe Announces Adobe Tag Manager for the Online Marketing Suite"
aimatch.com         # "Ad Server, SAS® Intelligent Advertising for Publishers"
analytics.edgesuite.net
atedra.com          # "Atedra est un réseau de publicité Internet francophone au Canada"
axf8.net            # https://www.eff.org/deeplinks/2013/06/third-party-resources-nsa-leaks
betrad.com          # "Evidon: Home | Online Marketing Intelligence, Web Analytics, Privacy" (which also publishes "Ghostery" add-on..)
bizographics.com    # "Business Audience Marketing"
bkrtx.com

# "Imagine Having The Power To Turn Abandoning Visitors Into Customers"
# "The BounceX software is tracking all the cursor movements of every visitor in real-time" (yikes!)
bounceexchange.com

clicktale.com       # "See absolutely everything your visitors do on your webpage ... See their every mouse move, click and keystroke"
clicktale.net       # Redirect to `clicktale.com`
crosspixel.net      # (cookies, localStorage) "leading provider of high performance audience data and information for the real-time advertising industry"
crsspxl.com         # Related to crosspixel.net
datarating.com      # see https://github.com/gorhill/httpswitchboard/issues/343 
displaymarketplace.com
erovinmo.com        # No info whatsoever from site itself can be found = naughty corner. Ironically spotted at "http://www.technologyreview.com/news/519336/bruce-schneier-nsa-spying-is-making-us-less-safe/" (also: http://www.mywot.com/en/scorecard/erovinmo.com)
exelator.com        # "domain used by eXelate Media which is an advertising company that is part of a network of sites, cookies, and other technologies used to track you" (Ref.: http://www.donottrackplus.com/trackers/exelator.com.php)
everestjs.net       # related to `everesttech.net`
everesttech.net     # "search engine marketing (SEM) solutions", pixel image on the page, looks like tracking to me. Spotted @ `http://www.homedepot.ca/` (search worked fine when blocking this hostname)
eyereturn.com       # "eyeReturn Marketing is the only end-to-end digital advertising platform in the market"
gigya.com           # "The tools you need to connect with consumers, harness rich data, and make marketing relevant"
inmuiads.info       # 
janrainbackplane.com # "Easily visualize, segment and update customer profiles to enable true personalized marketing"
krxd.net            # https://www.eff.org/deeplinks/2013/06/third-party-resources-nsa-leaks
lijit.com           # "We provide online advertising services, audience analytics"
llnwd.net           # http://en.wikipedia.org/wiki/Limelight_Networks
lduhtrp.net
mathtag.com         # "domain used by MediaMath to place cookies, on behalf of its customers, on the computers of visitors to our selected customer's websites and who may view our customer's display advertisements"
mxpnl.com           # 
moatads.com         # https://www.eff.org/deeplinks/2013/06/third-party-resources-nsa-leaks
mookie1.com         # "Specializing in online digital advertising, search marketing"
msads.net           # Sounds like ads, and no home web page...
omtrdc.net          # Redirect to Omniture
outbrain.com        # https://www.eff.org/deeplinks/2013/06/third-party-resources-nsa-leaks
panoramtech.net     # As seen in a screenshot at http://arstechnica.com/security/2014/01/malware-vendors-buy-chrome-extensions-to-send-adware-filled-updates/
parsely.com         # http://en.wikipedia.org/wiki/Parse.ly
peer39.net          # https://www.eff.org/deeplinks/2013/06/third-party-resources-nsa-leaks
pub2srv.com         # "This url is used by ad network Propeller Ads Media for ad serving"
servebom.com        # no home page, seen as 'tracking.servebom.com': good enough for this list

# These have "tracking" in domain name...
tracking.tomsguide.com
tracking.tomshardware.com
tracking.tomshardware.co.uk

wunderloop.net      # https://www.eff.org/deeplinks/2013/06/third-party-resources-nsa-leaks
yceml.net

# Blacklist maintained by HTTP Switchboard
# Prevent Facebook from tracking you

# If there is more to add, ask for a pull request

facebook-web-clients.appspot.com
facebook.com
facebook.net
fbcdn.net
fbstatic-a.akamaihd.net
```

**Comment by gorhill** [comment]
This could be related to #53, behavior is the same.

Meanwhile, I used the element picker to create a cosmetic filter:

`deviantart.com##.dp-ad-chrome.atf-right-300x250.dp-ad-visible`

Seems to work. You may try this until I figure a real fix for the bug.

**Comment by gorhill** [comment]
Actually, this one works better, you should not even see the placeholder at all:

`deviantart.com##.dp-ad-chrome.atf-right-300x250`

**Comment by qwerty12** [comment]
Awesome. Thank you very much.

**Comment by gorhill** [comment]
Fixed in [0.2.3.2](https://github.com/gorhill/uBlock/releases/tag/0.2.3.2).
